src/LIBSVM.jl

```diff
@@ -420,19 +393,15 @@ function svmpredict(model::SVM{T}, X::AbstractMatrix{U}; nt::Integer = 0) where
         decvalues = zeros(Float64, dcols, ninstances)
     end
 
-    ccall((:svm_set_print_string_function, libsvm), Cvoid,
-          (Ptr{Cvoid},), @cfunction(svmnoprint, Cvoid, (Ptr{UInt8},)))
+    libsvm_set_verbose(false)
 
     cmod, data = svmmodel(model)
-    ma = [cmod]
 
     for i = 1:ninstances
         if model.probability
-            output = ccall((:svm_predict_probability, libsvm ), Float64, (Ptr{Cvoid}, Ptr{SVMNode}, Ptr{Float64}),
-                            ma, nodeptrs[i], pointer(decvalues, nlabels*(i-1)+1))
+            output = libsvm_predict_probability(cmod, nodeptrs[i], decvalues[:, i])
         else
-            output = ccall((:svm_predict_values, libsvm ), Float64, (Ptr{Cvoid}, Ptr{SVMNode}, Ptr{Float64}),
-                            ma, nodeptrs[i], pointer(decvalues, nlabels*(i-1)+1))
+            output = libsvm_predict_values(cmod, nodeptrs[i], decvalues[:, i])
```

Comment:
I think `decvalues[:, i]` will make a copy.

maybe something like this?
```suggestion
            output = libsvm_predict_values(cmod, nodeptrs[i], Ref(decvalues, nlabels * (i - 1) + 1))
```

Reply:
True, it will make a copy. My intention was to avoid the explicit index computation. But maybe it is the recommended way to pass a pointer to a matrix's column? Or maybe `@view decvalues[:, i]` would work?

Btw. we should add a test for decision values too :)

Reply:
I'm not sure about the `view`.

> Btw. we should add a test for decision values too :)

yeah, we need.